app/hyperterm.js

```diff
@@ -89,7 +91,8 @@ export default class HyperTerm extends Component {
         }</div>
       </div>
       <div className={classes('resize-indicator', { showing: this.state.resizeIndicatorShowing })}>
-        { this.state.cols }x{ this.state.rows }
+        <div>{ this.state.fontSize }px</div>
```

Comment:
ideally, we only show the font size change when the font size changes, and the cols and rows when the cols and rows change